core/utilities/mixins/_flex-items-per-row.scss

```diff
@@ -5,24 +5,25 @@
 @mixin flex-items-per-row
 (
   $items: 1,
-  $selector: '*',
+  $parent: '&',
+  $children: '> *',
   $center-grid: true,
   $equal-heights: true,
   $grid: $neat-grid
 ) {
   $_gutter: _retrieve-neat-setting($grid, gutter);
 
   @if ($center-grid == true) and ($equal-heights == true) {
-    @include flex-container($justify-content: center, $align-items: stretch, $selector: $selector);
+    @include flex-container((justify-content: center, align-items: stretch, parent: $parent, children: $children));
```

Comment:
Here, we are creating a map on the fly with only the values that we want to override from the default container. I debated between this and creating a map for 'default-centered-equal-height-container' - but that felt too heavy and specific.